ci-operator/step-registry/hypershift/kubevirt/install/disconnected/hypershift-kubevirt-install-disconnected-commands.sh

```diff
@@ -0,0 +1,267 @@
+oc -n openshift-config extract secret/pull-secret --to="/tmp" --confirm
+set +x
+mirror_token=\$(cat "/tmp/.dockerconfigjson" | jq -r --arg var1 "\${mirror_registry}" '.auths[\$var1]["auth"]'|base64 -d)
+skopeo login "\${mirror_registry}" -u "\${mirror_token%:*}" -p "\${mirror_token#*:}"
+QUAY_USERNAME=openshift-cnv+openshift_ci
+QUAY_PASSWORD=\$(cat /home/openshift_cnv_pullsecret)
+skopeo login -u "\$QUAY_USERNAME" -p "\$QUAY_PASSWORD" quay.io/openshift-cnv
+set -x
+
+CNV_PRERELEASE_CATALOG_IMAGE=\$(cat /home/cnv-prerelease-catalog-image)
+CNV_PRERELEASE_VERSION=\$(cat /home/cnv-prerelease-version)
+# TODO: handle stable when needed
+#CNV_CHANNEL="stable"
+CNV_CHANNEL="nightly-\${CNV_PRERELEASE_VERSION}"
+
+echo "4: skopeo copy docker://\${CNV_PRERELEASE_CATALOG_IMAGE} oci:///home/cnv-local-catalog --remove-signatures"
+skopeo copy "docker://\${CNV_PRERELEASE_CATALOG_IMAGE}" "oci:///home/cnv-local-catalog" --remove-signatures
+
+echo "5. extract oc-mirror from image oc-mirror:v4.13.9"
+set +x
+QUAY_USER=\$(cat "/home/registry_quay.json" | jq -r '.user')
+QUAY_PASSWORD=\$(cat "/home/registry_quay.json" | jq -r '.password')
+skopeo login quay.io -u "\${QUAY_USER}" -p "\${QUAY_PASSWORD}"
+set -x
+oc_mirror_image="quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:278b6167e214992b2a40dd2fb44e8588f4a9ef100a70ec20cada58728350dd02"
+oc image extract \$oc_mirror_image --path /usr/bin/oc-mirror:/home --confirm
+if ls /home/oc-mirror >/dev/null ;then
+    chmod +x /home/oc-mirror
+else
+    echo "Warning, can not find oc-mirror abort !!!"
+    exit 1
+fi
+
+echo "6. oc-mirror --config /home/imageset-config.yaml docker://\${mirror_registry} --oci-registries-config=/home/registry.conf --continue-on-error --skip-missing"
+catalog_image="openshift-cnv/nightly-catalog" # TODO: handle stable when needed
+
+cat <<END |tee "/home/registry.conf"
+[[registry]]
+ location = "registry.redhat.io/rhacm2"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "quay.io:443/acm-d"
+    insecure = true
+[[registry]]
+ location = "registry.redhat.io/multicluster-engine"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "quay.io:443/acm-d"
+    insecure = true
+[[registry]]
+ location = "registry.access.redhat.com/openshift4/ose-oauth-proxy"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "registry.redhat.io/openshift4/ose-oauth-proxy"
+    insecure = true
+[[registry]]
+ location = "registry.stage.redhat.io"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "brew.registry.redhat.io"
+    insecure = true
+[[registry]]
+ location = "registry-proxy.engineering.redhat.com/rh-osbs"
+ insecure = true
+ blocked = false
+ mirror-by-digest-only = false
+ [[registry.mirror]]
+    location = "brew.registry.redhat.io/rh-osbs"
+    insecure = true
+END
```

Comment:
There are many unnecessary configurations in installing KubeVirt, such as `registry.redhat.io/rhacm2`, `registry.redhat.io/multicluster-engine`, etc.